vividus-reportportal-agent/build.gradle

```diff
@@ -0,0 +1,26 @@
+project.description = 'Vividus agent for Report Portal'
+
+repositories {
+    maven {
+        url 'https://jitpack.io'
```

Comment:
The JAR of the agent-java-jbehave is already in place (https://jitpack.io/#reportportal/agent-java-jbehave/c3303855e1), so that will be used for all subsequent builds, but I'm not sure about retention period of jar artifacts, there is no info on faq/documentation pages